src/simpla-img.html

```diff
@@ -121,6 +125,13 @@
        */
       _styleEditable(editable) {
```

Comment:
I'd rename this function given it's now doing two things - or just split into two functions? As is though it's a bit confusing given it does something definitely not to do with styling